app/scripts/controller-init/controller-list.ts

```diff
@@ -52,5 +66,6 @@ export type ControllerFlatState = AccountsController['state'] &
   PPOMController['state'] &
   PreferencesController['state'] &
   SmartTransactionsController['state'] &
+  SnapController['state'] &
```

Comment:
Why are we not passing the InsightsController and InterfaceController state type here ?

Reply:
Forgot to add them, fixed!